integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/MetalakePageTest.java

```diff
@@ -74,7 +84,7 @@ public void testCreateMultipleMetalakes() {
 
     for (int i = 0; i < towPagesCount; i++) {
```

Comment:
What mean is `towPagesCount`?

Reply:
The default counts of one page is 10, I need create over 11 counts of metalakes to make the `Go to next page` available.